.travis.yml

```
@@ -23,6 +23,16 @@ matrix:
     env: TOXENV=py36-django18 DEPLOY=1
   - python: "3.7"
     env: TOXENV=py37-django18
+  - python: "2.7"
+    env: TOXENV=py27-django111
+  - python: "3.4"
+    env: TOXENV=py34-django111
+  - python: "3.5"
+    env: TOXENV=py35-django111
+  - python: "3.6"
+    env: TOXENV=py36-django111 DEPLOY=1
```

This shouldn't have DEPLOY=1, that enables the "release to PyPI" step after tagging.  Only one test configuration should have that set as there is no point having more than one trying to do the release (and I guess some possibility of errors if two are trying to release at the same time?)  So you should either remove it here, or remove the other instance of it a few lines above.

```suggestion
    env: TOXENV=py36-django111
```

---

Fixed.